tests/testthat/test-add_nevent.tbl_uvregression.R

```diff
@@ -0,0 +1,80 @@
+skip_if_not(is_pkg_installed(c("broom.helpers", "broom"), reference_pkg = "gtsummary"))
+
+test_that("add_nevent.tbl_regression() works", {
+  expect_silent(
+    tbl <-
+      tbl_uvregression(
+        trial,
+        y = response,
+        method = glm,
+        method.args = list(family = binomial),
+        include = c(trt),
+        exponentiate = TRUE,
+        hide_n = TRUE
+      )
+  )
+  # total N added to table is accurate
+  expect_error(
```

Comment:
We can use `expect_silent()` here to ensure there are no unwanted errors, warnings, or messages.